algorithms/src/Kokkos_Sort.hpp

```diff
@@ -606,8 +613,20 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+template <class ExecutionSpace, class DataType, class... Properties>
+std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
+                 (SpaceAccessibility<
+                     HostSpace, typename Kokkos::View<DataType, Properties...>::
+                                    memory_space>::accessible)>
+sort(const ExecutionSpace&, const Kokkos::View<DataType, Properties...>& view) {
```

Comment:
I am not sure about that.  Why are you forwarded to `std::sort` for other execution spaces than `Serial`.
`Serial` is the only case where this is a no-brainer.